app/assets/javascripts/home.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
i think we can definitely remove this one -- if / when we decide to add some front end code, I'm guessing we won't use coffeescript